solarnet/roles/ipfs/templates/config.j2

```diff
@@ -51,30 +51,14 @@
     "Writable": false,
     "HTTPHeaders": {
       "Access-Control-Allow-Origin": [
-        "https://gateway.ipfs.io",
-        "https://ipfs.io",
-        "http://gateway.ipfs.io",
-        "http://ipfs.io",
-        "http://h.ipfs.io",
-        "http://localhost",
-        "http://localhost:8080",
-        "http://127.0.0.1",
-        "http://127.0.0.1:8080"
+        "*"
       ]
     }
   },
   "API": {
     "HTTPHeaders": {
       "Access-Control-Allow-Origin": [
-        "https://gateway.ipfs.io",
-        "https://ipfs.io",
-        "http://gateway.ipfs.io",
-        "http://ipfs.io",
-        "http://h.ipfs.io",
-        "http://localhost",
-        "http://localhost:8080",
-        "http://127.0.0.1",
-        "http://127.0.0.1:8080"
```

Comment:
Ooof this api/gateway duality in go-ipfs needs reworking.
On Fri, Nov 20, 2015 at 18:22 Lars Gierth <notifications@github.com> wrote:

> In solarnet/roles/ipfs/templates/config.j2
> <https://github.com/ipfs/infrastructure/pull/109#discussion_r45539900>:
>
> >        ]
> >      }
> >    },
> >    "API": {
> >      "HTTPHeaders": {
> >        "Access-Control-Allow-Origin": [
> > -        "https://gateway.ipfs.io",
> > -        "https://ipfs.io",
> > -        "http://gateway.ipfs.io",
> > -        "http://ipfs.io",
> > -        "http://h.ipfs.io",
> > -        "http://localhost",
> > -        "http://localhost:8080",
> > -        "http://127.0.0.1",
> > -        "http://127.0.0.1:8080"
>
> sorry for the delay here --
>
> we should only change this for the Gateway, not the API
>
> The API headers rules apply to :8080/api, which is a bit inconvenient
> here -- maybe we should just change that in go-ipfs?
>
> if port gets exposed to outer host -- docker firewall stopping?
>
> No firewalling, I made docker and nginx expose :5001 and :8080 only to
> 127.0.0.1 and the cjdns VPN
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ipfs/infrastructure/pull/109/files#r45539900>.
>